editoast/src/schema/v2/scenario.rs

```diff
@@ -0,0 +1,44 @@
+use chrono::NaiveDateTime;
+use serde_derive::{Deserialize, Serialize};
+use utoipa::ToSchema;
+
+use crate::modelsv2::{Project, Study};
+
+#[derive(Debug, Deserialize, Serialize, ToSchema)]
+#[aliases(ScenarioV2 = Scenario)]
+pub struct ScenarioV2 {
+    pub id: i64,
+    pub infra_id: i64,
+    pub name: String,
+    pub description: String,
+    pub creation_date: NaiveDateTime,
+    pub last_modification: NaiveDateTime,
+    pub tags: Vec<String>,
+    pub timetable_id: i64,
+    pub study_id: i64,
+}
```

Comment:
You can remove this struct as it is not used

Reply:
It is utilized for OpenAPI. We currently have both Scenario (v1) and Scenario (v2). When we generate OpenAPI, the V2 replaces the V1. To address this, I have renamed it to ScenarioV2 to prevent conflicts.